Source/WebKit/UIProcess/API/C/WKWebsiteDataStoreConfigurationRef.h

```diff
@@ -84,6 +84,9 @@ WK_EXPORT void WKWebsiteDataStoreConfigurationSetStaleWhileRevalidateEnabled(WKW
 WK_EXPORT WKStringRef WKWebsiteDataStoreConfigurationCopyPCMMachServiceName(WKWebsiteDataStoreConfigurationRef configuration);
 WK_EXPORT void WKWebsiteDataStoreConfigurationSetPCMMachServiceName(WKWebsiteDataStoreConfigurationRef configuration, WKStringRef name);
 
+WK_EXPORT bool WKWebsiteDataStoreConfigurationGetResourceLoadStatisticsDebugModeEnabled(WKWebsiteDataStoreConfigurationRef configuration);
```

Comment:
Please don't add new C API unless absolutely necessary, and I don't think it is in this case.